src/modules/wallets/sagas.js

```diff
@@ -27,7 +27,7 @@ const checkRegistration = function * ({ blockchainKey, userID }) {
 // Check all blokchain registrations
 const checkRegistrations = function * () {
   try {
-    const userID = yield select(S.login.userID)
+    const userID = 'current' // yield select(S.login.userID)
```

Comment:
This does not really make sense + could let to mistakes.

When no login, userID should be null, and not set from sagas.

What you can do is put userID = null as initial state in redux state. And here still use the yield select. And then having the code handling both cases userID = 2234234 (when logged) or null.